modules/download_models.py

```diff
@@ -0,0 +1,21 @@
+# download_models.py
+import json
+import os
+
+from modules.model_loader import load_file_from_url
+
+
+def download_models(url, selected, file_name=None):
+    with open('./model_config_path.json', 'r') as f:
+        model_paths = json.load(f)
```

Comment:
the content of this file must be configurable using the config settings. Please remove model_config_path.json and use config settings instead as combined array here.
If you run into circular imports then you'll have to rearrange the code accordingly.